Add Antrea as one of the CNI solutions in kubeadm (#22477)
kubernetes/website: content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

{{< tabs name="tabs-pod-install" >}}

{{% tab name="Antrea" %}}
[Antrea](https://github.com/vmware-tanzu/antrea) is an [Open vSwitch](https://www.openvswitch.org/) based networking and network policy solution. Open vSwitch enables Antrea to implement Kubernetes Network Policies in a high-performance and efficient manner.

> In order Antrea to detect POD network range, you must pass `--pod-network-cidr` parameter to `kubeadm init`.

Reviewer: This line isn't idiomatic English.

Maybe:
```suggestion
You must pass `--pod-network-cidr` parameter to `kubeadm init`. Antrea relies on this configuration.
```

(the thing I'm least keen on is “POD”, which is correct in terms of language but not right in terms of page style).

Reviewer: One reason we're keen on readable English is to reduce the effort for localization teams when picking up changes.

> In order Antrea to detect POD network range, you must pass `--pod-network-cidr=192.168.0.0/16` to `kubeadm init`.

Reviewer: This wording implies that only 192.168.0.0/16 is supported. I assume that Antrea wouldn't make that restriction, as it would be very limiting.

```suggestion
{{< note >}}
If you're using Antrea, you configure a Pod network range by adding a `--pod-network-cidr` parameter to `kubeadm init`.
{{< /note >}}
```

(also, Pod is written “Pod” not “POD”).

Author: Thanks @sftim  yes it makes sense, I will change it in a second!
